deepfence_server/reporters/scan/notify.go

```diff
@@ -88,8 +88,8 @@ func injectNodeData[T any](results []T, common model.ScanResultsCommon,
 			flag := integration.IsMessagingFormat(integrationType)
 			if flag {
 				ts := m["updated_at"].(int64)
-				tm := time.Unix(0, ts*int64(time.Millisecond))
-				m["updated_at"] = tm
+				tm := time.Unix(0, ts*int64(time.Millisecond)).In(time.UTC)
+				m["updated_at"] = tm.Format("02-01-2006 15:04:05 MST")
```

Comment:
Shouldn't this come from the system timezone currently used by the user ?? @varunsharma0286

Reply:
@shyam-dev We are sending the timestamps in UTC for the below integrations:
Slack, Teams, PagerDuty, Email and Jira. 
On the UI, we are displaying the timeformat in the user's timezone.